frontend/src/store/wallet.ts

```diff
@@ -243,6 +244,24 @@ export default function useWalletStore() {
       ]);
       const _isAccountSetup = _stealthKeys !== null;
 
+      if (typeof _userEns === 'string') { // ENS address must exist
+        const chainId = provider.value.network.chainId;
+        switch (chainId) {
+          case 1: // mainnet
+          case 137: // polygon mainnet
+            avatar.value = await MAINNET_PROVIDER.getAvatar(_userEns);
+            break;
+          case 4: // rinkeby
+            const currentNetworkAvatar = await provider.value.getAvatar(_userEns);
+            const mainnetAvatar= await MAINNET_PROVIDER.getAvatar(_userEns);
```

Comment:
Running `yarn lint` in the frontend returns errors on these two lines:

```
/Users/mds/Documents/projects/umbra/umbra-protocol/frontend/src/store/wallet.ts
  255:13  error    Unexpected lexical declaration in case block  no-case-declarations
  256:13  error    Unexpected lexical declaration in case block  no-case-declarations
```

From https://eslint.org/docs/rules/no-case-declarations:

> This rule disallows lexical declarations (`let`, `const`, `function` and `class`) in `case`/`default` clauses. The reason is that the lexical declaration is visible in the entire switch block but it only gets initialized when it is assigned, which will only happen if the case where it is defined is reached.
>
> To ensure that the lexical declaration only applies to the current case clause wrap your clauses in blocks.

So, let's just wrap the case blocks in braces

Reply:
Actually, a few lines up you'll see `lookupEnsName` where we pass a mainnet provider, i.e. we show the mainnet name even when connected on Rinkeby. Therefore we should probably be consistent and not special-case Rinkeby here. Which means we can probably get rid of the whole switch-case and just always use mainnet resolution. 

The current switch-case approach also means we'd need to add another step to adding support for additional networks

Reply:
> Running yarn lint in the frontend returns errors

My bad. I checked for typescript errors but forgot we used eslint too

> a few lines up you'll see lookupEnsName where we pass a mainnet provider, i.e. we show the mainnet name even when connected on Rinkeby

Ahh, yeah, good point. I'll just get rid of the switch 👍